src/static/css/infusion-docs.css

```diff
@@ -351,24 +347,81 @@ div.infusion-docs-note {
     min-width: 13rem;
 }
 
+@media only screen and (max-width: 50rem) {
```

Comment:
I have no idea where this came from. I didn't create or edit it. I will investigate. If it's there for mobile purposes, I'll probably remove it.